Sources/SwifterAuth.swift

```diff
@@ -76,28 +53,39 @@ public extension Swifter {
                    failure: FailureHandler? = nil) {
         let callbackURLScheme = callbackURL.absoluteString.components(separatedBy: "://").first
         self.postOAuthRequestToken(with: callbackURL, success: { token, response in
-            var requestToken = token!
             let queryURL = self.makeQueryURL(tokenKey: token!.key, forceLogin: forceLogin)
-            let session = ASWebAuthenticationSession(url: queryURL, callbackURLScheme: callbackURLScheme) { (url, error) in
+            self.session = ASWebAuthenticationSession(url: queryURL, callbackURLScheme: callbackURLScheme) { (url, error) in
```

Comment:
If you're storing the session, make sure this isn't causing a retain cycle.

Reply:
I fixed it to set `session` to nil after the callback by commit 6d65fb8.